app/js/event/event.sort.js

```diff
@@ -7,53 +7,45 @@
 
   eventSort.$inject = ['$http'];
 
-  // function eventSort($http) {
-  //
-  //     var getEvents = function() {
-  //         return $http.get('data/mock.events.json');
-  //     };
-  //
-  //     return {
-  //         getEvents: getEvents
-  //     };
-  //
-  // };
-
-  function getDistanceFromLatLonInKm(lat1,lon1,lat2,lon2) {
-    var R = 6371; // Radius of the earth in km
-    var dLat = deg2rad(lat2-lat1); // deg2rad below
-    var dLon = deg2rad(lon2-lon1);
-    var a =
-      Math.sin(dLat/2) * Math.sin(dLat/2) +
-      Math.cos(deg2rad(lat1)) * Math.cos(deg2rad(lat2)) *
-      Math.sin(dLon/2) * Math.sin(dLon/2);
-    var c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1-a));
-    var d = R * c; // Distance in km
-    return d;
-  };
+  function eventSort($http) {
+
+    var vm = this;
```

Comment:
This is a factory function though. So I don't think it refers to where you would like it to refer to :wink:
When you have tested this factory you probably have some code that hooks this into the other parts of the code base. Could you also push those changes here so it's easier to try this out?